deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -480,7 +480,14 @@ async def get_interactive_waveform_data(
             download_directory_from_s3(bucket=PULSE3D_UPLOADS_BUCKET, key=key, file_path=tmpdir)
 
             # read the time force dataframe from the parquet file
-            parquet_path = glob(os.path.join(tmpdir, "time_force_data", "*.parquet"), recursive=True)
+            # older versions of pulse3d to not inlcude the version data
+            if hasattr(parsed_meta, "version"):
+                parquet_path = glob(
+                    os.path.join(tmpdir, "time_force_data/", parsed_meta["version"], "*.parquet"),
```

Comment:
The point of using `os.path.join` is so that you don't need to include a `/` or `\`, this function will determine the correct slash to use for you, so we can remove it here